cmd/buildctl/build/oci.go

```diff
@@ -0,0 +1,21 @@
+package build
+
+import (
+	"fmt"
+
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/content/local"
+)
+
+// ParseOCILayouts parses --oci-layout
+func ParseOCILayouts(layoutDirs []string) (map[string]content.Store, error) {
```

Comment:
Might be better to skip this for now and concentrate on LLB tests. This value can't be just a dir name anyway. You need to find a specific image and its digest from that dir.

Reply:
Done and removed. Now I wonder, we added `opt.OCILayouts` as you suggested, but how do we then populate it?

Reply:
OK, completely ignored the `--oci-layout` option, focused on the `--opt context:foo=` path for now.